components/dashboard/src/user-settings/Preferences.tsx

```diff
@@ -83,6 +94,11 @@ export default function Preferences() {
                     </a>
                 </Subheading>
                 <SelectIDE location="preferences" />
+                <Heading2 className="mt-12">Reset Start Options</Heading2>
+                <Subheading>Forget all previously cached options for repositories and projects.</Subheading>
+                <Button className="mt-4" type="secondary" onClick={clearCachedStartWorkspaceOptions}>
+                    Clear cached start options
```

Comment:
**nitpick**: Minor button label rephrase to be shorter and more concise, also avoiding scary terms like cache.

```suggestion
                    Clear Defaults
```